bundles/org.openhab.binding.resol/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="resol"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<bridge-type id="vbuslan">
+		<label>Bridge VBusLAN adapter</label>
+		<description>This bridge represents the Resol VBus-LAN adapter
+		</description>
+
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<context>network-address</context>
+				<label>IP Address</label>
+				<description>The IP address of the of the VBus-LAN adapter.</description>
+			</parameter>
+			<parameter name="port" type="integer" required="false" min="1024" max="65535">
+				<context>network-address</context>
+				<label>VBus-LAN adapter live data port</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions